postgresql-17.yaml

```diff
@@ -195,6 +200,68 @@ subpackages:
       provides:
         - libpq=${{package.full-version}}
 
+  - name: plpython3
+    description: PostgreSQL Python libraries
+    dependencies:
```

Comment:
is melange already adding the appropriate `python3.13`-specific dependencies? (I imagine `so:libpython3.13` or similar?). See below comment in testing